sdk/Cargo.toml

```diff
@@ -24,6 +24,7 @@ bmff = []  # Work in progress support for BMFF-based containers
 file_io = ["openssl"]
 serialize_thumbnails = []
 xmp_write = ["xmp_toolkit"]
+no_interleaved_io = ["file_io"]
```

Comment:
I'm not sure we need this feature, I suppose it is possible someone might want to build a single threaded version, but I think it is more likely that we will want to add threading in more places.

Reply:
It may be the case for some environments.